tests/contract/test_research_paper_parser_chunker.py

```
@@ -68,3 +73,62 @@ def test_markdown_paper_parser_rejects_missing_file(tmp_path):
         MarkdownPaperParser().parse(str(missing_path))
 
     assert "论文文件不存在" in str(exc_info.value)
+
+
+def _run_sample_script(args: list[str], cwd: Path) -> subprocess.CompletedProcess[str]:
+    """! @brief 执行 sample 脚本并强制超时，避免 CI 卡死。"""
```

Ruff (0.15.15)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)